sqladmin/models.py

```diff
@@ -786,8 +791,8 @@ def _url_for_action(self, request: Request, action_name: str) -> str:
             )
         )
 
-    def _get_pk(self, obj: Any) -> Any:
-        return getattr(obj, get_primary_key(obj).name)
+    def get_identifier(self, obj: Any) -> Any:
```

Comment:
Isn't this redundant? Can't we use the `get_object_identifier` in all places instead?

Reply:
Yes that's true. I did this mainly to mirror the original code's `_get_pk` function and because this function is being used inside of some of the HTML templates. ([sqladmin/templates/details.html](https://github.com/aminalaee/sqladmin/pull/507/files#diff-bc54d17435a60b52d052a84ddfc98cd65333343f2ff639d100d1a2028302d315L68), sqladmin/templates/list.html)

Should I get rid of this method and make `get_object_identifier` available in the templates like so?:

```patch
diff --git a/sqladmin/application.py b/sqladmin/application.py
index 8adf907..0eb71a4 100644
--- a/sqladmin/application.py
+++ b/sqladmin/application.py
@@ -92,6 +92,7 @@ class BaseAdmin:
         templates.env.globals["zip"] = zip
         templates.env.globals["admin"] = self
         templates.env.globals["is_list"] = lambda x: isinstance(x, list)
+        templates.env.globals["get_object_identifier"] = get_object_identifier
 
         return templates
 ```

Reply:
Hm..yeah that's ok, let's keep it as you have already.